dashboard/app/models/activity_section.rb

```diff
@@ -31,7 +31,7 @@ class ActivitySection < ApplicationRecord
   belongs_to :lesson_activity
   has_one :lesson, through: :lesson_activity
 
-  has_many :script_levels, -> {order(:activity_section_position)}
+  has_many :script_levels, -> {order(:activity_section_position)}, dependent: :destroy
```

Comment:
WARNING: removing a script level from an activity section will destroy it, and losing script levels in production would be bad because then we would lose the teacher feedback associated with them. 

This shouldn't be a problem for us right now, because no script levels in production should be in activity sections. but it's worth keeping in mind because it means there is no such thing as "temporarily" putting a script level in an activity section.

Reply:
So if an editor goes in make an activity with a script level saves it and then waits for it to deploy and then plays with it on production and then decides to delete that activity on levelbuilder will that cause problems?

Reply:
I'm not really sure I can say if there is a specific problem here. It might not be fair to call this a land mine. but maybe this is a sharp edge and the teacher feedbacks (because they depend on script level ids in production) are all balloons 😬 

there's really no way a curriculum writer could get into a bad state here right now because of the protection we have in place with `disallow_legacy_script_levels`, but it's possible that this in combination with something we build in the future could become a problem.

I think what would make me feel better about this is to get that "legacy" script flag set up sooner rather than later, so that we have a harder boundary preventing any existing courses from being modified by the new editing tools.

Reply:
Should we also be looking to get teacher feedbacks off script level?

Reply:
I think so, yes. I opened this item: [PLAT-403]